src/main/java/de/tum/in/www1/artemis/web/rest/ModelingExerciseResource.java

```diff
@@ -198,6 +199,27 @@ public ModelingExerciseResource(ModelingExerciseRepository modelingExerciseRepos
         }
     }
 
+    /**
+     * Return the Compass statistic regarding the automatic assessment of the modeling exercise with the given id.
+     *
+     * @param exerciseId the id of the modeling exercise for which we want to get the Compass statistic
+     * @return the statistic as key-value pairs in json
+     */
+    @GetMapping("/exercises/{exerciseId}/compass-statistic")
+    @PreAuthorize("hasAnyRole('ADMIN')")
+    public ResponseEntity<Void> getCompassStatisticForExercise(@PathVariable Long exerciseId) {
+        ModelingExercise modelingExercise = modelingExerciseService.findOne(exerciseId);
```

Comment:
Misses error handling for non existing modeling exercise.

Reply:
The error handling is done by the service method.